t/rose-metadata-check/09-custom-macro.t

```diff
@@ -30,7 +30,7 @@ init <<__META_CONFIG__
 [namelist:macro_nl=my_macro_var1]
 macro=envswitch.LogicalTransformer
 __META_CONFIG__
-init_macro envswitch.py < $TEST_SOURCE_DIR/lib/custom_macro.py
+init_macro envswitch.py < "$TEST_SOURCE_DIR"/lib/custom_macro.py
```

Comment:
nicer?
```suggestion
init_macro envswitch.py < "${TEST_SOURCE_DIR}/lib/custom_macro.py"
```

Reply:
I didn't even do this, looks like I need to fiddle the settings for the shellchecker extension for VSCode